dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -189,4 +189,6 @@ void connectUsingFile(String filePath) throws SQLException, IOException,
             FileNotFoundException;
 
     DbStoredProcedureCall newStoredProcedureCall(String name, DbParameterAccessor[] accessors);
+
+    String buildFunctionCall(String procOrFuncName, int numberOfParameters);
```

Comment:
The existing/proposed changes to `DBEnvironment` were in line with what we'd done previously, especially for Informix, where we allowed environment-specific function call construction/execution.